profiles/lib/profile_v2.lua

```diff
@@ -0,0 +1,9 @@
+-- profile api version 2
```

Comment:
Is there a functional reason for having this as an own file, or was this just a refactor?

I would be cool if we could make the dependencies explicit in each profile.